assets/questions/rle.js

```diff
@@ -78,7 +85,10 @@ def DecodeEncodedString(word):
     performanceTests: []
   }, {
     instructions: [
-      'You need to make sure that your code handles short strings correctly.'
+      [
+        'Next, double-check your code to make sure it handles short strings. ',
+        'Ideally, these strings should also be as small as possible.',
```

Comment:
The use of two different adjectives suggests two different things though. If you want them to mean the same thing, maybe just use "short" throughout?